components/runtime/chezscheme/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Richard Lowe
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= configure
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		chezscheme
+COMPONENT_VERSION=	9.5.2
+COMPONENT_SUMMARY=	Chez Scheme
+COMPONENT_PROJECT_URL=	https://github.com/cisco/ChezScheme
+COMPONENT_FMRI=		runtime/chezscheme
+COMPONENT_CLASSIFICATION=Development/Other Languages
+COMPONENT_SRC=		ChezScheme-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+  sha256:3a370fdf2ffd67d6a0ccbb993dfab1cbaf4a0a97983c869cfaab40528c33c48b
+COMPONENT_ARCHIVE_URL= \
+  https://github.com/cisco/ChezScheme/archive/v$(COMPONENT_VERSION).tar.gz
+
+COMPONENT_LICENSE_FILE=	$(COMPONENT_SRC)/LICENSE
```

Comment:
It's not found if it's just `COMPONENT_LICENSE_FILE=LICENSE`? The build system looks for the file in few directories, it should be detected in `COMPONENT_SRC`.